crates/egui-winit/src/lib.rs

```diff
@@ -852,138 +860,125 @@ fn open_url_in_browser(_url: &str) {
     }
 }
 
-/// Winit sends special keys (backspace, delete, F1, …) as characters.
-/// Ignore those.
-/// We also ignore '\r', '\n', '\t'.
-/// Newlines are handled by the `Key::Enter` event.
-fn is_printable_char(chr: char) -> bool {
-    let is_in_private_use_area = '\u{e000}' <= chr && chr <= '\u{f8ff}'
-        || '\u{f0000}' <= chr && chr <= '\u{ffffd}'
-        || '\u{100000}' <= chr && chr <= '\u{10fffd}';
-
-    !is_in_private_use_area && !chr.is_ascii_control()
-}
-
-fn is_cut_command(modifiers: egui::Modifiers, keycode: winit::event::VirtualKeyCode) -> bool {
-    (modifiers.command && keycode == winit::event::VirtualKeyCode::X)
+fn is_cut_command(modifiers: egui::Modifiers, keycode: winit::keyboard::KeyCode) -> bool {
+    (modifiers.command && keycode == winit::keyboard::KeyCode::KeyX)
         || (cfg!(target_os = "windows")
             && modifiers.shift
-            && keycode == winit::event::VirtualKeyCode::Delete)
+            && keycode == winit::keyboard::KeyCode::Delete)
 }
 
-fn is_copy_command(modifiers: egui::Modifiers, keycode: winit::event::VirtualKeyCode) -> bool {
-    (modifiers.command && keycode == winit::event::VirtualKeyCode::C)
+fn is_copy_command(modifiers: egui::Modifiers, keycode: winit::keyboard::KeyCode) -> bool {
+    (modifiers.command && keycode == winit::keyboard::KeyCode::KeyC)
```

Comment:
I think this should probably be `winit::keyboard::Key::Character("c".into())` instead of being based on keycodes, so that any user keymap is taken into account.

Reply:
Yeah, I think in general we should probably use `logical_key` instead of `physical_key`

Reply:
I've created an issue to better track this: https://github.com/emilk/egui/issues/3653